microsoft-edge/origin-trials/index.md

```diff
@@ -1,26 +1,204 @@
+
+1. Click the **Submit** button.
+
+   An origin trial token is generated, and is displayed in the **Registrations** section near the **New Origin Trial Registration** form:
+
+   ![The generated token displayed in the Registration section](./index-images/token-displayed.png)
+
+1. Click the **Copy** button, and then paste the origin trial token into a safe place, noting which domain it's for.  You can also view this information in this webpage later.
+
+
+<!-- ====================================================================== -->
+## Use the origin trial token at your website
+<!-- probably don't change wording of heading, b/c the "Enable the Origin Trial in your site's code" card at https://developer.microsoft.com/microsoft-edge/origin-trials probably links to this heading/anchor -->
+
+Use the token at your website in either of the following ways:
+
+
+<!-- ------------------------------ -->
+#### Option 1: A \<meta\> tag within the \<head\> element of a webpage
+
+Within the `<head>` element of a webpage, include the following `<meta>` tag:
+
+`<meta http-equiv="origin-trial" content="EXAMPLE_TOKEN">`
+
+Replace `EXAMPLE_TOKEN` by your token.
+
+
+<!-- ------------------------------ -->
+#### Option 2: An HTTP header in your HTTP server response
+
+In your HTTP server response, add the following HTTP header:
+
+`Origin-Trial: EXAMPLE_TOKEN`
+
+Replace `EXAMPLE_TOKEN` by your token.
+
+
+<!-- ------------------------------ -->
+#### Perform feature detection, and provide graceful fallback
+
+In your website code, perform feature detection.  See [Implementing feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) at MDN.
+
+In your website code, provide graceful fallback mechanisms in case the origin-trial token expires, or in case the Microsoft Edge feature team ends the origin trial.  Treat origin trials as experimental.
+
+
+<!-- ------------------------------ -->
+#### Test the origin-trial feature at `localhost`
+
+To test an origin-trial feature at `localhost`, go to `edge://flags` and turn on the appropriate feature flag.  Microsoft Edge origin trials only support SSL-enabled domains, not `localhost`.
+
+
+<!-- ------------------------------ -->
+#### Opt out of an origin trial experiment
+
+To opt out of an experiment (origin trial) that you're enrolled in, remove the origin trial token from the `<meta>` tag or from the server response headers.
+
+
+<!-- ====================================================================== -->
+## Renew an origin trial token
+
+Origin trial tokens expire in 6 weeks, by default<!-- todo: strike ", by default"? -->.  When the origin trial token expires, you must renew the token, which means generating a fresh, new token for this origin trial.  You can renew the token before it expires.
```

Comment:
strike "by default"?  raises unresolved questions.  if we keep "by default", need to add hint or explanation of what scenario would be other than the default.  
are we saying that some tokens expire in 4 weeks or 8 weeks, depending on feature team?
@leahmsft